src/handle-checkout-mutation.js

```diff
@@ -1,8 +1,14 @@
 export default function handleCheckoutMutation(mutationRootKey, client) {
-  return function({data, model}) {
+  return function({data, errors, model}) {
     const rootData = data[mutationRootKey];
     const rootModel = model[mutationRootKey];
-    const userErrors = rootData.userErrors;
+    let userErrors = [];
+
+    if (errors != null && errors.length) {
```

Comment:
This doesn't necessarily mean a mutation "failed". The mutation can succeed but an error could be caused by a return field. Depends what you want, but you could check if `rootData` is `null` as well.